reviewer-guide.en.md

```diff
@@ -44,10 +45,29 @@ There are four different vote options: -1, -0, +0, +1.
 
 Each vote option's meaning is as follows:
 * **-1**：Strong Reject    This **SHOULD NOT** be in PyCon TW 2025
-* **-0**：Weak Reject
-* **+0**：Weak Accept
+* **-0**：Weak Reject    This **IS ACCEPTABLE, BUT SLIGHTLY NOT RECOMMENDED** to be in PyCon TW 2025 
+* **+0**：Weak Accept    This **IS ACCEPTABLE, AND SLIGHTLY RECOMMENDED** to be in PyCon TW 2025
 * **+1**：Strong Accept    This **SHOULD** be in PyCon TW 2025
 
+# Reviewer's Responsibility and Review Rate
+Dear Reviewers,
+Thank you very much for agreeing to serve as a reviewer for this conference. Your professional contributions are essential to ensuring the technical quality of the conference. We understand that you have a busy schedule and other personal commitments. However, your participation plays a critical role in enhancing the quality and fairness of the conference.
+
+In order to ensure a smooth review process and to guarantee that each proposal is reviewed in a timely and thorough manner, we hope to achieve the following common goals:
+
+### 1. Expectations for Review Rate
+We encourage each reviewer to take responsibility for reviewing proposals in fields they are familiar with. The ideal review rate should reach at least 80% for the proposals within your area of expertise. This will ensure that most proposals receive a sufficient number of reviews, providing appropriate feedback to the authors and ensuring the quality of the conference program.
+
+We understand that each reviewer’s schedule may vary, but we greatly appreciate your efforts in completing the review tasks as much as possible. Your dedication will play a crucial role in the smooth running of the conference and the enhancement of the program's quality.
+
+### 2. The Importance of Review Quality
+In addition to the quantity of reviews, we also emphasize the quality of reviews. Each proposal should be evaluated based on its technical value and contributions to open-source or other relevant aspects. We encourage you to provide detailed feedback and constructive suggestions, which will help authors improve their proposals and ensure that the conference showcases high-level research results.
+
+### 3. In Case You Cannot Review on Time
+If, for any reason, you are unable to complete a review on time due to scheduling or other issues, please inform us as soon as possible. We will do our best to assist you. You can contact us at program@python.tw.
+
+### 4. Thank You for Your Support and Understanding
+We truly appreciate the time you have dedicated to reviewing proposals amidst your busy work or personal schedule. Your contribution is invaluable, and we sincerely thank you for your support. We look forward to making this conference a smooth and outstanding event with your help.
```

Comment:
Put this inside the section `# Reviewer's Responsibility and Review Rate` looks weird to me. As it's apparently not reviewer's responsibility or rate